atlp-rwanda/elite-bn-fe
# chore(setup-material-UI):Initialize a material UI theme to use for the app

#### What does this PR do?
Initialize a material UI theme to use for the app.
It should have fonts defined and other necessary general stylings.

#### How should this be manually tested?

1. Clone this repository
2. Switch to this branch  ch-setup-materialUI-#182923295
3. Install dependencies by running `yarn` command
4. Start the server; `yarn start`
5. You should be able to see the menu now and when you click to `themes` you can see more details about the UI theme, font, and other stylings.

#### Any background context you want to provide?

The color code used was provided on the UI/UX Figma design.

#### What are the relevant pivotal tracker stories?
[#182923295](https://www.pivotaltracker.com/story/show/182923295)
